sagemaker_neo_compilation_jobs/gluoncv_yolo_darknet/train_yolo.py

```diff
@@ -524,22 +432,31 @@ def model_fn(model_dir):
     :param: model_dir The directory where model files are stored.
     :return: a model (in this case a Gluon network)
     """
+    logging.info('Invoking user-defined model_fn')
+
+    import neomxnet
```

Comment:
can we instead have the following:
`import neomxnet  # noqa: F401`

the comment aside import will prevent linters to remove the import as this is not being directly used code-wise but it indirectly required and used.

Reply:
Thanks, just added that in the following commit.